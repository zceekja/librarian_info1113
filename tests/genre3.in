add book book1.csv 111115
add book book1.csv 111116
genre Comedy