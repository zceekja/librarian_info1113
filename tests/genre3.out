user: Successfully added: The Hitchhiker's Guide to the Galaxy (Douglas Adams).

user: Successfully added: So Long and Thanks for all the Fish (Douglas Adams).

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: 